files/jsons/Policies/appID_Management/ptu_2.json

```diff
@@ -0,0 +1,1277 @@
+{

```

Comment:
@dboltovskyi  Updated policy table should contain sections: module_config, functional_groupings, consumer_friendly_messages, and app_policies.
See https://adc.luxoft.com/confluence/display/APPLINK/Policies+-+appID+Management
 AppLink Policies Manager Specification v1.3.doc
Also check chapter 10.3 Section of the Policy Table, that you have correct elements in the sections. You should be sure that SDL will validate and apply this table.